README.md

```diff
@@ -23,6 +23,11 @@ access to all of its features. Your target system must have a functional [SDL2 l
 * macOS: `brew install sdl2`
 * Ubuntu Linux / Debian: `sudo apt install libsdl2-dev`
 
+Install [RISCOF](https://riscof.readthedocs.io/en/stable/installation.html#install-riscof).
+```shell
+pip3 install riscof
```

Comment:
Check https://github.com/sysprog21/rv32emu/blob/master/.ci/riscv-tests.sh for RISCOF installation. We shall instal `riscof` from git repository for the sake of latest development.